fedimint-ui/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "printWidth": 80,
+  "semi": true,
+  "trailingComma": "es5",
```

Comment:
It might not matter much since this is just style but in `fedimint-ui/packages/tsconfig/react-app.json` we have it set to es6.

Reply:
def don't need this rule if we are targetting es anyways